examples/websocket-relay/relay-app/src/lib/workflow.ts

```diff
@@ -132,90 +158,83 @@ export function fail(workflowId: WorkflowId) {
 
 // HANDLER
 
-export async function handleMessage(event: MessageEvent) {
-  const data = await event.data.text();
-
+export async function handleMessage(data: MaybeResult) {
```

Comment:
this should be `MaybeResult<Schemas.WorkflowNotification, Schemas.WorkflowNotificationError>` so you get the proper types all the way down i will export more types from the entry point and release a new version

Reply:
Updated with the switch to `@fission-codes/homestar` 1.0.0.